test/test_save_load.jl

```diff
@@ -32,9 +51,9 @@ import JustPIC._3D as JP3
     @test eltype(eltype(JP2.Array(Float32, phase_ratios).vertex.data)) === Float32
 
     jldsave(
-        "particles.jld2"; 
-        particles    = JP2.Array(particles), 
-        phases       = JP2.Array(phases), 
+        "particles.jld2";
+        particles    = JP2.Array(particles),
```

Comment:
```suggestion
        particles    = Array(particles),
```
And else where, `JP2.` or `JP3.` is not really needed